README.md

```diff
@@ -45,3 +45,75 @@ Newsletter Component for Koala Framework
             ...
             - { path: ^/api/v1/subscribers, roles: IS_AUTHENTICATED_ANONYMOUSLY }
             ...
+
+#### Subscriber Open API
+
+* Disable CSRF protection for open api in `config.yml`
```

Comment:
Instead you could inject this setting by yourself like here: https://gitlab.vivid-planet.com/vivid/kwf-oauth2-server/blob/2.0/DependencyInjection/KwfOAuth2ServerExtension.php

Reply:
It's could be done partially.  (CSRF) config changes can be injected but changes of access_control and adding firewalls in security could not be done in a bundle:

https://github.com/symfony/symfony/issues/22308